### PR TITLE
Enforce Lint in all modules, fix violation, update AGP 3.1.1 → 3.2.0-alpha15.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**

### CODE REVIEW

File: gradle/lint.gradle [hunk 1]
```diff
@@ -0,0 +1,30 @@
+def domicLintOptions() {
+    return {
+        warningsAsErrors true
+        abortOnError true
```
Comment (vanniktech) [inline gradle/lint.gradle]: That's the default

File: gradle/lint.gradle [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+        textReport true
+        textOutput 'stdout'
+
+        htmlReport true
```
Comment (vanniktech) [inline gradle/lint.gradle]: This one too

File: gradle/lint.gradle [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        xmlReport true
```
Comment (vanniktech) [inline gradle/lint.gradle]: Same

File: gradle/lint.gradle (hunk 1 above)
Comment (artem-zinnatullin) [reply gradle/lint.gradle]: Just being explicit here, you know my habits haha

**Comment by artem-zinnatullin** [comment]
I'll leave explicit values as is @vanniktech, sorry :)

**Pull request merged**